doc/source/whatsnew/v1.0.0.rst

```diff
@@ -191,6 +191,16 @@ method on a :func:`pandas.api.indexers.BaseIndexer` subclass that will generate
 indices used for each window during the rolling aggregation. For more details and example usage, see
 the :ref:`custom window rolling documentation <stats.custom_rolling_window>`
 
+.. _whatsnew_1000.groupby_key:
+
+Allow NaN in groupby key
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+We've added a ``dropna`` keyword to :meth:`DataFrame.groupby` and :meth:`Series.groupby` in order to
+allow ``NaN`` values in group keys. Users can define ``dropna`` to ``False`` if they want to include
+``NaN`` values in groupby keys. The default is set to ``True`` for ``dropna`` to keep backwards
+compatibility (:issue:`3729`)
```

Comment:
add an example (like you have in the doc-strings)

also add examples in groupby.rst (and point from here)

Reply:
added in both!